packages/terra-popup/src/terra-dev-site/doc/popup/Popup.1.doc.jsx

```diff
@@ -34,18 +38,22 @@ const DocPage = () => (
       {
         title: 'Popup Display With Arrow',
         example: <PopupWithArrow />,
+        source: PopupWithArrowSrc,
```

Comment:
`<PopupWithArrow />` needs to have detail for enabling arrow.

[this](https://github.com/cerner/terra-framework/blob/9fcea5284e7c4b7621b3675249f8d0dadd47e064/packages/terra-popup/src/terra-dev-site/doc/common/PopupCommonExample.jsx#L17) should default to `false` and other examples consuming PopupCommonExample should adjust accordingly.

Reply:
Fixed here https://github.com/cerner/terra-framework/pull/927/commits/1e41be63db4d72a46f315d694ebded26ff6f5b68